raster3d/r3.in.v5d/v5d.c

```diff
@@ -2336,7 +2366,8 @@ int v5dWriteCompressedGrid(const v5dstruct *v, int time, int var,
     pos = grid_position(v, time, var);
     if (lseek(v->FileDesc, pos, SEEK_SET) < 0) {
         /* lseek failed, return error */
-        printf("Error in v5dWrite[Compressed]Grid: seek failed, disk full?\n");
+        G_warning(
+            _("Error in v5dWrite[Compressed]Grid: seek failed, disk full?\n"));
```

Comment:
Should have at least the `%s"), strerror(errno));` part and leave out `\n` (G_warning calls don't have `\n` at the end).

What was you idea here @nilason? Keep the original error message or use the same one as elsewhere?

Reply:
I tried to express my opinion in https://github.com/OSGeo/grass/pull/4141#discussion_r1767647769, I’d say same as in the other cases. I see no reason to keep the original message.

Reply:
Done changes

Reply:
@ShubhamDesai I'm pretty sure @nilason meant to make it same and replace the original message, i.e.:

```c
G_warning(_("Unable to seek: %s"), strerror(errno));
```

Reply:
Done